src/util/storage/drivers/Example.hpp

```diff
@@ -59,55 +96,9 @@ namespace opentxs::storage
  */
 class StorageExample : public virtual Plugin, public virtual Driver
 {
-private:
+public:
```

Comment:
all methods below are overrided so maybe better will be use "protected" (except destructor)?

Reply:
sure